### PR TITLE
Add documentation on URL Normalization

### PR DESCRIPTION
**Pull request by pedrosousa**

### CODE REVIEW

File: products/rules/src/content/normalization/manage.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    </Aside>
+
+The table below shows how URL Normalization settings affect incoming URLs before they pass to other Cloudflare features and to origin.
+
+<TableWrap>
+
+Incoming URL | Normalize incoming URLs | Normalize URLs to origin | URL passed to Cloudflare edge | URL passed to origin
+---|---|---|---|---
+`www.example.com/hello`   | _On_  | _Off_ | `www.example.com/hello`   | `www.example.com/hello`
```
Comment (abracchi-tw) [inline products/rules/src/content/normalization/manage.md:50]: I think this table is a great idea, it helps me understand the concept really well. One question: what was the thought process behind showing what happens to `www.example.com/hello`, which never undergoes any transformation in any of the setups?

File: products/rules/src/content/normalization/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Cloudflare provides a URL Normalization feature to modify the URLs of incoming requests so that they conform to a consistent formatting standard.
+
+When you enable URL Normalization, all incoming URLs are normalized before they pass to subsequent Cloudflare edge features that accept a URL input, such as Page Rules, Firewall Rules, Workers, and Access. Rule expressions that filter traffic based on URLs will therefore trigger correctly, regardless of the format of the incoming URL. When URL Normalization is disabled, Cloudflare forwards the URL to origin in its original form.
+
+## How it works
+
+Cloudflare URL Normalization is similar to [rfc3986](https://www.ietf.org/rfc/rfc3986.txt), and modifies separators, encoded elements, and literal bytes in incoming URLs as follows:
+
+* The following unreserved characters are percent decoded:
```
Comment (abracchi-tw) [inline products/rules/src/content/normalization/index.md]: this is me not being familiar with the language - I was a bit confused by `percent decoded` and the fact that the bullet points are structured (as far as I understand) to show the result of the normalization first, and the characters that get decoded last. feel free to disregard this as I'm literally 0 expert on this!

**Review by abracchi-tw — APPROVED** [review]

File: products/rules/src/content/normalization/manage.md (hunk 1 above)
Comment (pedrosousa) [reply products/rules/src/content/normalization/manage.md:50]: To let readers know that `/` characters are left untouched in "regular" URLs.

File: products/rules/src/content/normalization/index.md (hunk 2 above)
Comment (pedrosousa) [reply products/rules/src/content/normalization/index.md]: Yes, you're right! I think it's useful to list the affected characters first, though. I'll add `decoded from` right before the percent-encoded version.

Comment (pedrosousa) [reply products/rules/src/content/normalization/index.md]: I also added a link to the RFC section that explains percent encoding.

**Pull request merged**